test/sql/pragma/test_show_tables.test

```diff
@@ -11,19 +11,43 @@ CREATE TABLE "select"(i INTEGER);
 statement ok
 CREATE VIEW v1 AS SELECT DATE '1992-01-01' AS k
 
+statement ok
+CREATE TABLE t1 (id INTEGER PRIMARY KEY, j VARCHAR)
+
+statement ok
+CREATE TABLE t2 (id INTEGER PRIMARY KEY, j VARCHAR UNIQUE)
+
+# column_name | column_type | null | key | default | extra
+query TTTITI
+DESCRIBE t1
+----
+id	INTEGER	NO	PRI	NULL	(empty)
+j	VARCHAR	YES	(empty)	NULL	(empty)
```

Comment:
`DESCRIBE` is modeled after MySQL, here is what MySQL outputs for this specific table:

```
mysql> CREATE TABLE t2 (id INTEGER PRIMARY KEY, j VARCHAR(100) UNIQUE);
Query OK, 0 rows affected (0.01 sec)

mysql> DESCRIBE t2;
+-------+--------------+------+-----+---------+-------+
| Field | Type         | Null | Key | Default | Extra |
+-------+--------------+------+-----+---------+-------+
| id    | int          | NO   | PRI | NULL    |       |
| j     | varchar(100) | YES  | UNI | NULL    |       |
+-------+--------------+------+-----+---------+-------+
2 rows in set (0.00 sec)

```

Reply:
Thanks for comments I will undo some changes and push it here